autoload/asyncomplete/sources/tags.vim

```diff
@@ -91,7 +91,8 @@ function! s:lines_to_matches(matches, lines) abort
             let l:splits = split(l:line, "\t")
             if len(l:splits) > 0
                 let l:word = l:splits[0]
-                let a:matches[l:word] = 1
+                let l:type = l:splits[-1]
+                call add(a:matches, {"word": l:word, "menu": "[tag: " . l:type . "]"})
```

Comment:
should we also allow ignore case and dup?

Reply:
Added -- also added `[tag]` to the results from `getcompletion()`